build.gradle

```diff
@@ -65,7 +65,7 @@ android {
 
 dependencies {
     implementation 'com.amplitude:analytics-connector:1.0.0'
-    implementation 'com.squareup.okhttp3:okhttp:4.2.2'
+    implementation 'com.squareup.okhttp3:okhttp:4.12.0'
```

Comment:
LGTM, but a [heads up in case we start getting new support requests](https://github.com/amplitude/Amplitude-Android/issues/309) for `Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl` or similar.